lib/pages/chat/events/message.dart

```diff
@@ -180,9 +180,8 @@ class Message extends StatelessWidget {
                                                       AppConfig
                                                           .bubbleSizeFactor,
                                             ),
-                                      constraints: const BoxConstraints(
-                                        maxWidth:
-                                            FluffyThemes.columnWidth * 1.5,
+                                      constraints: BoxConstraints(
+                                        maxWidth: MediaQuery.of(context).size.width * 0.8,
```

Comment:
IMO: Should use `Expanded(flex: ...)`, it good for this case

Reply:
you are right it's more reliable